lessons/r/ggplot2/ggplot2.Rmd

```diff
@@ -395,6 +294,148 @@ p + theme_bw()
 p + theme_classic()
 ```
 
+However, it's much more likely you'll want to create a theme specific
+to your own needs. Fortunately, saving custom themes is quite straightforward.
+Let's save the above theme to an object:
+
+```{r}
+fig_theme <- theme(axis.title = element_text(family = 'Helvetica', size = 14), 
+        axis.text = element_text(family = 'Helvetica', size = 14, color = 'black'),
+        axis.line = element_line(color = 'black', size = 0.75),
+        panel.background = element_blank(),
+        legend.key = element_blank())
+```
+
+This can now be added to plots like any other `ggplot` object:
+
+```{r}
+p +
+  fig_theme
+```
+
+We'll be revisiting another use of this concept later in the lesson.
+
+#### Legend formatting
+
+Legend formatting involves a few extra considerations. First of all,
+if you want to remove a legend altogether, this is done with `guides()`:
+
+```{r}
+p +
+    theme(axis.title = element_text(family = 'Helvetica', size = 14), 
+        axis.text = element_text(family = 'Helvetica', size = 14, color = 'black'),
+        axis.line = element_line(color = 'black', size = 0.75),
+        panel.background = element_blank(),
+        legend.key = element_blank()) +
+  guides(color = FALSE)
+```
+
+Notice that the argument given the `guides` is the aesthetic (in this
+case, color) that we were using to generate the legend. 
+
+Let's move our legend around and style the text a little:
+
+```{r}
+p +
+    theme(axis.title = element_text(family = 'Helvetica', size = 14), 
+        axis.text = element_text(family = 'Helvetica', size = 14, color = 'black'),
+        axis.line = element_line(color = 'black', size = 0.75),
+        panel.background = element_blank()) +
+  theme(legend.key = element_blank(),
+        legend.position = 'bottom', # special arg - values: 'top', 'left', etc
+        legend.title = element_text(size = 12),
+        legend.text = element_text(size = 12))
+```
+
```

Comment:
Might be useful to list here some of the other plot elements that can be modified using `theme()` just to give folks a flavour of what's possible.

Reply:
Good call - I added `axis.text.x` and make reference to several other possible changes throughout the lesson now (i.e. facet labels, plot tags).